modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -68,8 +68,11 @@ https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4
 == What are the limits on an object store?
 
 The total size of an object store is not limited, but each value is limited to 10 MB.
-The free version also limits usage to 10 transactions per second (TPS) per app.
-Premium add-on users are allowed up to 100 TPS per app.
+
+Every API call (via the connector or API) to Object Store v2 counts toward the transactions per second (TPS) limit:
```

Comment:
I think It might be more clear if we include "rate limit" term in our tps. Something like

Every API call (via the connector or API) to Object Store v2 counts toward  API rate limiting (transactions per second (TPS)):

Reply:
fixed